recipes/r-boutroslabplottinggeneral/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: boutroslabplottinggeneral
+  version: "5.3.4"
+
+source:
+  fn: BoutrosLab.plotting.general_5.3.4.tar.gz
+  url: http://bpg.oicr.on.ca/src/contrib/BoutrosLab.plotting.general_5.3.4.tar.gz
+
+requirements:
+  build:
+      - r-base
+      - r-rcolorbrewer
+      - r-lattice
+      - r-latticeextra
+      - r-r.methodss3
+      - r-hexbin
+      - r-proto
+      - r-findpython
+      - r-getopt
+      - r-rjson
+      - r-argparse
+      - r-assertthat
+      - r-r6
+      - r-rcpp
+      - r-lazyeval
+      - r-tibble
+      - r-magrittr
+      - r-dbi
+      - r-bh
+      - r-dplyr
+
+  run:
+      - r-base
```

Comment:
You probably need all these dependencies above also as run dependency. Please also include a test, otherwise we can not merge it.